nemo/collections/nlp/modules/common/token_classifier.py

```diff
@@ -67,7 +67,6 @@ def __init__(
         )
         self.post_init(use_transformer_init=use_transformer_init)
 
-    @typecheck()
```

Comment:
why is this removed?

Reply:
Depending on `self.log_softmax`, `TokenClassifier` returns either `LogProbsType` or `LogitsType`. I was thinking about removing type check until we have the option of returning one of several types. What do you think about it?